03_hardware_configuration/slides/slides.md

```diff
@@ -0,0 +1,631 @@
+﻿
+Agenda
+
+| ID | Topic |
+| -- | ----- |
+| 00 | Introduction to the workshop |
+| 01 | Introduction to AX Code IDE |
+| 02 | Get started with your first AX Project |
+| **03** | **Hardware Configuration** |
+| 04 | Introduction to ST Programming |
+| 05 | Loading and Debugging |
+| 06 | OOP Elements of ST |
+| 07 | Unit Testing |
+| 08 | Tools for commissioning |
+| 09 | Package management |
+
+
+---
+
+<header class="slide_header">
+  <h2>Prerequisites</h2>
+</header>
+
+<div class="grid-two-col-eq">
+  <div class="flex-col justify-center">
+    <p>To get started, you need to have SIMATIC AX, apax and all its prerequisites installed.</p>
+    <p>In addition you should know the basics on how to use AX Code as well as have a starting project and know the basics of hardware configuration.</p>
+    <br/>
+    <p>With this you are set up to continue with this learning path.</p>
+  </div>
+</div>
+
+---
+
+<header class="slide_header">
+  <h2>
+        What will you learn in this chapter
+  </h2>
+</header>
+
+  <div class="flex-col justify-center">
+    <p>After you complete this training section, you will have:</p>
+        <ul>
+            <li>Basic knowledge of hardware configuration</li>
+            <li>Know how to insert hardware devices into a project</li>
+            <li>Know how to use templates in hardware context</li>
+            <li>Be able to administer users through User Management</li>
+            <li>Know how to perform hardware basic setting assignment</li>
+            <li>Understand diagnostics in the hardware environment</li>
+        </ul>
+    <br>
+</div>
+
+---
+
+<header class="slide_header">
+    <h2>IT-Like Hardware Engineering</h2>
+    <h3>The description of the configuration</h3>
+</header>
+
+<div class="grid-two-col-eq">
+    <div class="flex-col justify-center">
+        <p>
+        In order to create hardware for a PLC, the first step we need to take is to understand how the hardware configuration is described in a textual declaration for an IT-like context.</p>
```

Comment:
You're not "creating hardware", "hardware configuration" would be a better wording 

```suggestion
        In order to create the hardware configuration for a PLC, the first step we need to take is to understand how the hardware configuration is described in a textual declaration for an IT-like context.</p>
```